hack/update-1fmt.sh

```diff
@@ -1,63 +0,0 @@
-#!/bin/bash
-#
-# Copyright 2017 the Velero contributors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-set -o errexit
-set -o nounset
-set -o pipefail
-
-if [[ ${1:-} == '--verify' ]]; then
-  # List file diffs that need formatting updates
-  MODE='-d'
-  ACTION='Verifying'
-else
-  # Write formatting updates to files
-  MODE='-w'
-  ACTION='Updating'
-fi
-
-if ! command -v goimports > /dev/null; then
-  echo 'goimports is missing - please run "go get golang.org/x/tools/cmd/goimports"'
-  exit 1
-fi
-
-files="$(find . -type f -name '*.go' -not -path './.go/*' -not -path './vendor/*' -not -path './site/*' -not -path '*/generated/*' -not -name 'zz_generated*' -not -path '*/mocks/*')"
-echo "${ACTION} gofmt"
-output=$(gofmt "${MODE}" -s ${files})
-if [[ -n "${output}" ]]; then
-  VERIFY_FMT_FAILED=1
-  echo "${output}"
-fi
-if [[ -n "${VERIFY_FMT_FAILED:-}" ]]; then
-  echo "${ACTION} gofmt - failed! Please run 'make update'."
```

Comment:
Please add a replacement for `  echo "${ACTION} gofmt - failed! Please run 'make update'."` that was previously visible via github actions log.

You can add this to curent linter check action by [setting continue-on-error: true](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error)

Then you would add [status check function](https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions) in the next step and force a failure if prior step fail, but this time you echo "Please run `make lint-fix`"

Reply:
did not understand the request. Do you want to add a message saying what to do if this action https://github.com/vmware-tanzu/velero/actions/runs/7462814485/job/20306039678?pr=7194 fails?

Reply:
```yaml
    - name: developer instruction
      if: ${{ failure() }}
      run: echo "Please fix linter issues. Run `make lint-fix` to help you"
```

Reply:
Sure. Yup.

Reply:
alright, will try if my example works

Reply:
` echo "Please fix linter issues. Run `make lint-fix` to help you" && exit 1`

Reply:
added instruction in documentation, we can add this in follow up PR

since this PR is more than 3 months old, I want to just change things absolute necessary in it

if you do not mind, I will resolve this one